samples/dfu/single_slot/Kconfig.sysbuild

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config MCUBOOT_SIGNATURE_USING_KMU
```

Comment:
Note: this is needed, as there is no simple way to add "soc specific sysbuild.conf" file. Thus, we resort to a Kconfig.sysbuild to set nRF54L specific `SB_CONFIG` options